RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -73,6 +76,12 @@
     vertexSrc      = 'firstStepGoodPrimaryVertices',
     howToUseMeasurementTracker = 'Never'
 ))
+jetCoreRegionalStepBarrelTrackingRegions = jetCoreRegionalStepTrackingRegions.clone(RegionPSet=dict(
+    JetSrc = 'jetsForCoreTrackingBarrel',
+))
```

Comment:
this is used only by DQM and not by reco.
Is there a rationale to monitor products that are not used in reco?